controllers/machinedeployment_controller_test.go

```diff
@@ -389,6 +389,21 @@ var _ = Describe("MachineDeployment Reconciler", func() {
 			return len(machineSets.Items)
 		}, timeout*5).Should(BeEquivalentTo(0))
 
+		By("Verifying MachineDeployment has correct Conditions")
```

Comment:
we are not using ginkgo anymore here, please rebase atop main branch.

Reply:
Done. Thanks for pointing that out!